open-api.yaml

```diff
@@ -0,0 +1,1614 @@
+      required:
+        - hits
+        - offset
+        - limit
+        - nbHits
+        - exhaustiveNbHits
+        - processingTimeMs
+        - query
+      title: ''
+    update:
+      type: object
+      description: |
+        MeiliSearch is an asynchronous API. It means that the API does not behave as you would typically expect when handling the request's responses.
+
+        Some operations are put in a queue and will be executed in turn (asynchronously). In this case, the server response contains the identifier to track the execution of the operation.
+
+        [Learn More](/learn/advanced/asynchronous_updates.html).
+      additionalProperties: false
+      properties:
+        status:
+          description: The state of the operation.
+          enum:
+            - enqueued
+            - processed
+            - failed
+        updateId:
+          type: integer
+          description: The id of the update.
+        type:
+          type: object
+          additionalProperties: false
+          required:
+            - name
+            - number
+          properties:
+            name:
+              type: string
+            number:
+              type: integer
+        enqueuedAt:
+          $ref: '#/components/schemas/timestamp'
+        duration:
+          type: number
+          description: The number of seconds taken to complete the operation.
+        processedAt:
+          $ref: '#/components/schemas/timestamp'
+        error:
+          type: string
+          description: |
+            A string describing [the error that occurred](/errors/).
+      required:
+        - status
+        - updateId
+        - type
+        - enqueuedAt
+    synonyms:
+      type: object
+      description: List of associated words treated similarly.
+      additionalProperties:
+        type: array
+        items:
+          type: string
+      example:
+        wolverine:
+          - xmen
+          - logan
+        logan:
+          - wolverine
+          - xmen
+        wow:
+          - world of warcraft
+    stopWords:
+      type: array
+      description: List of words ignored by MeiliSearch when present in search queries.
+      items:
+        type: string
+      example:
+        - of
+        - the
+        - to
+    rankingRules:
+      type: array
+      description: |
+        List of ranking rules sorted by order of importance.
+
+        [A list of ordered built-in ranking rules](/learn/core_concepts/relevancy.html#order-of-the-rules).
+      items:
+        type: string
+      example:
+        - typo
+        - words
+        - proximity
+        - attribute
+        - wordsPosition
+        - exactness
+        - asc(release_date)
+        - desc(rank)
+    filterableAttributes:
+      type: array
+      description: |
+        Attributes to use as [facets](/reference/features/faceted_search.html) and for [filtering](/reference/features/filtering.html).
+      items:
+        type: string
```

Comment:
what is item in this context?

Reply:
`items` is generated by Open API to say "this array attribute contains `string`".

        type: array
        items:
          type: string
        default: